api/v3/CiviShareTests/TestCiviMRF.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+function civicrm_api3_civi_share_tests_test_civi_m_r_f(&$params) {
```

Comment:
Long live APIv3? :wink:

Reply:
Just a workaround until the test infrastructure works … :roll_eyes: